src/actions.js

```diff
@@ -867,21 +867,21 @@ export const sharedAddressUpdate = (lastSharedAddress, lastSharedIndex) => ({
 });
 
 /**
- * Stores the walletConnect instance on the redux store
+ * Stores the reown instance on the redux store
  *
- * walletConnect {WalletConnect} The WalletConnect instance
+ * reown {Reown} The Reown instance
  */
-export const setWalletConnect = (walletConnect) => ({
+export const setReown = (reown) => ({
   type: types.SET_WALLET_CONNECT,
-  payload: walletConnect,
+  payload: reown,
 });
 
 /**
- * Dispatched with data when a WalletConnect QRCode is read
+ * Dispatched with data when a Reown QRCode is read
  *
- * data {string} The WalletConnect v2 URI
+ * data {string} The Reown v2 URI
```

Comment:
[question] The version API is still the same v2?

Reply:
No it's not, fixed! Thanks